Taskfile.yml

```diff
@@ -263,7 +263,7 @@ tasks:
     desc: Run {{.GENERATOR_APP}} lint checks.
     dir: '{{.GENERATOR_ROOT}}'
     cmds:
-      - golangci-lint run --verbose --fix --fast=false
+      - golangci-lint run --verbose --fix
```

Comment:
w/ no deadline in the file do we need to specify one everywhere?

Reply:
No, have put `timeout` in the file.